metrics/web/test/auth/presentation/pages/login_page_test.dart

```diff
@@ -63,6 +64,28 @@ void main() {
       },
     );
 
+    testWidgets(
+      "applies the theme brightness that corresponds the operating system's brightness",
```

Comment:
Can we also test that the Login page listens on the platform brightness somehow? I mean, the test case checking that if the platform brightness changes, the theme mode in the notifier changes as well.